pallets/nfts/src/tests.rs

```diff
@@ -953,17 +960,17 @@ fn set_item_metadata_should_work() {
 
 		// Successfully add metadata and take deposit
 		assert_ok!(Nfts::set_metadata(RuntimeOrigin::signed(account(1)), 0, 42, bvec![0u8; 20]));
-		assert_eq!(Balances::free_balance(&account(1)), 8);
+		assert_eq!(Balances::free_balance(&account(1)), 7); // free_balance - metadata deposit - item deposit - balance deposit
```

Comment:
To minimise the changes in the PR but illustrate what is happening:
```suggestion
		assert_eq!(Balances::free_balance(&account(1)), 8 - balance_deposit);
```